### ISSUE TITLE
Couldn't generate date at leap year

### ISSUE THREAD

**Issue by maxtar** [datafaker-net/datafaker#1102]
**Describe the bug**
When I try to generate birthday at 29 February (for exampe 29/02/2024) got exception:
```java
Invalid date 'February 29' as '1959' is not a leap year
java.time.DateTimeException: Invalid date 'February 29' as '1959' is not a leap year
	at java.base/java.time.LocalDate.create(LocalDate.java:457)
	at java.base/java.time.LocalDate.of(LocalDate.java:251)
	at java.base/java.time.LocalDateTime.of(LocalDateTime.java:261)
	at net.datafaker.providers.base.DateAndTime.birthday(DateAndTime.java:258)
	at net.datafaker.providers.base.DateAndTime.birthday(DateAndTime.java:243)
```

**To Reproduce**
```java
Faker faker = new Faker();
String s = faker.date().birthday();
```

**Expected behavior**
I don't know how resolve this situation the best way. Maybe return 28 february?

**Versions:**
 - OS: Windows 10
 - JDK: openjdk version "11.0.13" 2021-10-19 LTS
  OpenJDK Runtime Environment (build 11.0.13+8-LTS)         
  OpenJDK 64-Bit Server VM (build 11.0.13+8-LTS, mixed mode)
- Faker Version 1.9.0

**Comment by snuyanzin** [comment]
closing as duplicate https://github.com/datafaker-net/datafaker/issues/1098
it was fixed almost a half a year ago
please use 2.0.2 or 2.1.0+

**Comment by maxtar** [comment]
> closing as duplicate #1098 it was fixed almost a half a year ago at #954 please use 2.0.2 or 2.1.0+

We can't update to 2 version because we are use java 11.

**Comment by snuyanzin** [comment]
1.x is not supported
so probably need to ask whether @bodiam can release another fix version for 1.x
